app/views/stats/subscriptions.html.erb

```diff
@@ -2,6 +2,25 @@
     <h4> About this page </h4>
     <p> People subscribe to tags so that they can be updated when a research is posted with the tag they are subscribed to. This page shows the number of subscriptions and subscription rates on different tags. </p>
 </div>
+
 <div class="col-md-8 col-md-offset-1">
     <h2>Subscriptions</h2>
-</div>
+ 	<table class="table inline-grid">
+		<tr>
```

Comment:
Not a huge deal, but would you mind using just 2 spaces for indents? We've tried to standardize a bit on this to help keep our code tidier. Thanks, and sorry for the minute details! 😄

Reply:
yes of course. sorry I switched to a new editor that I'm still finding my way around. I'll fix that.